design-library/src/components/BccSpinner/BccSpinner.css

```diff
@@ -0,0 +1,4 @@
+@layer components {
+  .bcc-spinner {
```

Comment:
Currently this bcc-spinner css component is completely empty.

The guideline for creating components is to separate the css from the vue by building up this `bcc-spinner` class using `@apply` and Tailwind classed instead of using tailwind classes directly inside the vue template.